public/react-chat/app.jsx

```diff
@@ -1,5 +1,5 @@
 // A placeholder image if the user does not have one
-const PLACEHOLDER = '/placeholder.png';
+const PLACEHOLDER = 'https://placeimg.com/60/60/people';
```

Comment:
We can link to our own placeholder but it needs to be the full URL since those pages should be the same as the code in the guides.